### PR TITLE
Update rubygem-foreman-tasks to 0.16.1

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 1]
```diff
@@ -86,40 +82,12 @@ BuildRequires: npm(sass-loader) < 7.0.0
...
-BuildRequires: npm(seamless-immutable) >= 7.1.2
-BuildRequires: npm(seamless-immutable) < 8.0.0
-BuildRequires: npm(urijs) >= 1.19.1
-BuildRequires: npm(urijs) < 2.0.0
-BuildRequires: npm(uuid) >= 3.3.2
-BuildRequires: npm(uuid) < 4.0.0
+BuildRequires: npm(@theforeman/vendor) >= 0.1.1
+BuildRequires: npm(@theforeman/vendor) < 1.0.0
+BuildRequires: npm(humanize-duration) >= 3.20.1
+BuildRequires: npm(humanize-duration) < 4.0.0
```
Comment (tbrisker) [inline packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Looks like this module isn't packaged causing the build to fail

**Comment by tbrisker** [comment]
I think you need to do this in separate PRs so the package is available already when building

**Comment by ekohl** [comment]
[test rpm]

**Comment by ekohl** [comment]
Same c3 issue as 0.17.0:

```
BUILDSTDERR: ModuleNotFoundError: Module not found: Error: Can't resolve 'c3' in '/builddir/build/BUILDROOT/tfm-rubygem-foreman-tasks-0.16.1-1.fm1_23.el7.noarch/opt/theforeman/tfm/root/usr/share/gems/gems/foreman-tasks-0.16.1/webpack/ForemanTasks/Components/Chart'
```

**Comment by adamruzicka** [comment]
Same workaround until we get it fixed properly in next release

**Pull request merged**